apollo-portal/src/main/resources/static/i18n/en.json

```
@@ -446,6 +455,7 @@
   "Config.SortByKey": "Filter Config by Key",
   "Config.FilterConfig": "Filter",
   "Config.Reset": "Reset",
+  "Config.ClusterManage": "Cluster Manage",
```

It's better to use `Manage Cluster` so that it's in the same form as `Manage Project` and `Manage AccessKey`. 
```suggestion
  "Config.ManageCluster": "Manage Cluster",
```